src/SFA.DAS.ApplyService.Domain/Entities/Application.cs

```diff
@@ -29,6 +29,7 @@ public class ApplicationStatus
     {
         public const string InProgress = "In Progress";
         public const string Submitted = "Submitted";
+        public const string GatewayAssessed = "GatewayAssessed";
```

Comment:
Just for comment: GatewayStatus will be new column. I'm not sure how this will tie into ApplicationStatus yet, we'll probably need to look at a suitable workflow